bin/zeppelin-daemon.sh

```diff
@@ -94,7 +94,8 @@ function initialize_default_directories() {
     $(mkdir -p "${ZEPPELIN_PID_DIR}")
   fi
 
-  echo "Browse localhost:8080 in your browser"
+  IP_ADDR="$(ifconfig en0 | grep inet | grep -v inet6 | cut -d ' ' -f2)"
+  echo -e "Browse $IP_ADDR:8080 in your browser.\nIf you are testing on your local computer, use localhost:8080"
```

Comment:
@AhyoungRyu I updated script to read port num from zeppelin-env.sh, but still have problem on reading from zeppelin-site.xml. There are some xml parser but needs additional parser to be installed, and that seems really not good.
How do you think about adding id for each nodes? Like below...
```
<configuration>

<property>
  <name>zeppelin.server.addr</name>
  <value id="zeppelin.server.addr">0.0.0.0</value>
  <description>Server address</description>
</property>

<property>
  <name>zeppelin.server.port</name>
  <value id="zeppelin.server.port">>8080</value>
  <description>Server port.</description>
</property>
...
```
How do you think about it? This will cause no side effect as long as I know.
If you are not sure about it, I'll update this first so you could look through it carefully.